### ISSUE TITLE
Python 2.7.6 doesn't have itertools.zip_longest

### ISSUE THREAD

**Issue by key** [dtcooper/python-fitparse#15]
Hi guys.

I try to install fitparse from pypi and got error.
Did I need future module?

```
Downloading/unpacking fitparse
  Downloading fitparse-1.0.0.tar.gz
  Storing download in cache at /Users/key/.pip_cache/https%3A%2F%2Fpypi.python.org%2Fpackages%2Fsource%2Ff%2Ffitparse%2Ffitparse-1.0.0.tar.gz
  Running setup.py (path:/Users/key/Documents/virtualenv/gpslogparser/build/fitparse/setup.py) egg_info for package fitparse
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/Users/key/Documents/virtualenv/gpslogparser/build/fitparse/setup.py", line 4, in <module>
        import fitparse
      File "fitparse/__init__.py", line 1, in <module>
        from fitparse.base import FitFile, FitParseError
      File "fitparse/base.py", line 9, in <module>
        from fitparse.profile import FIELD_TYPE_TIMESTAMP, MESSAGE_TYPES
      File "fitparse/profile.py", line 7, in <module>
        from fitparse.records import (
      File "fitparse/records.py", line 3, in <module>
        from itertools import zip_longest
    ImportError: cannot import name zip_longest
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/Users/key/Documents/virtualenv/gpslogparser/build/fitparse/setup.py", line 4, in <module>

    import fitparse

  File "fitparse/__init__.py", line 1, in <module>

    from fitparse.base import FitFile, FitParseError

  File "fitparse/base.py", line 9, in <module>

    from fitparse.profile import FIELD_TYPE_TIMESTAMP, MESSAGE_TYPES

  File "fitparse/profile.py", line 7, in <module>

    from fitparse.records import (

  File "fitparse/records.py", line 3, in <module>

    from itertools import zip_longest

ImportError: cannot import name zip_longest
```

**Comment by key** [comment]
ah, is this fixed in ng HEAD?

**Comment by lukeremy** [comment]
I have the same issue. Is there a fix in the current version?

**Comment by pR0Ps** [comment]
Fixed as of 1cc59e64fe76ec26c6a588037a6e9a0678af714a
